src/components/Match/BuildingMap/buildingData.js

```diff
@@ -18,31 +18,31 @@ export default [
   },
   {
     id: 't1br',
-    style: { top: '83%', left: '78%' },
+    style: { top: '82%', left: '75%' },
```

Comment:
Hmm, changing these values directly may cause old matches to render improperly. Can we put the new values under a new var and use it conditionally?